lib/workbox/templates/sw.js

```diff
@@ -46,10 +46,17 @@ function initWorkbox(workbox, options) {
 
 function precacheAssets(workbox, options) {
   if (options.preCaching.length) {
-    workbox.precaching.precacheAndRoute(options.preCaching, options.cacheOptions)
+    const cacheOptions = options.cacheOptions || {};
+    const precacheList = options.preCaching.map(url => ({
+      url: url,
+      revision: cacheOptions.revision
+    }))
+
+    workbox.precaching.precacheAndRoute(precacheList, options.cacheOptions)
```

Comment:
According to google:
https://developers.google.com/web/tools/workbox/modules/workbox-precaching#explanation_of_the_precache_list

Revision must be passed separately per each precached file and not in options object.